lib/src/snapd_client.dart

```diff
@@ -1132,8 +1132,11 @@ class SnapdClient {
   /// If [query] is provided, searches for snaps that match the given string.
   /// If [name] is provided, match the snap with the given name.
   /// If [section] is provided, search within that store section.
+  /// If [select] is provided and set to:
+  ///   - 'refresh': search refreshable snaps. Can't be used with [query] nor [name].
+  ///   - 'private': search private snaps. Can't be used with [query].
   Future<List<Snap>> find(
-      {String? query, String? name, String? section}) async {
+      {String? query, String? name, String? section, String? select}) async {
```

Comment:
I think this would work better with either flags or an enumeration. i.e. you could do:
```dart
Future<List<Snap>> find({String? query, String? name, String? section, bool selectRefresh = false, bool selectPrivate = false})
```
and throw an assertion if both of the flags are set or:
```dart
enum SnapdFindSelect { refresh, private }
Future<List<Snap>> find({String? query, String? name, String? section, SnapdFindSelect? select})
```
In snapd-glib I made `SnapdFindFlags` for this, but it also included other flags, not just the select field.

Reply:
Thanks :+1:!
Went with `SnapdFindFilter` to match the naming convention of the other enums.